main.py

```diff
@@ -0,0 +1,58 @@
+from dotenv import load_dotenv
+import os
+import requests
+import pandas as pd
+import pymysql
+
+def main():
+	''' GET ENVIRONMENT VARIABLES AND SECRETS '''
+	load_dotenv() # Load .env
+	dbhost = os.getenv('DBHOST', "")
+	dbuser = os.getenv('DBUSER', "")
+	dbpassword = os.getenv('DBPASSWORD', "")
+	database = os.getenv('DATABASE', "")
+	table = os.getenv('TABLE', "")
```

Comment:
Talking in developer world, this one needs encapsulation.  It's okay to do this for single script,  but what in real world, we never have only single magic script.  Every scripts will have to change this part if connection have to be changed.